pkg/cmdutils/config.go

```diff
@@ -215,3 +220,19 @@ func (c *ClusterConfig) BookieClient() bookkeeper.Client {
 
 	return bk
 }
+
+func loadFromEnv() *ClusterConfig {
+	config := ClusterConfig{}
+	if envConf, ok := os.LookupEnv("PULSAR_CLIENT_CONF"); ok {
```

Comment:
It is a google idea to load existing pulsar client conf. I see there is a potential to wire this configuration setting into the new `context` command we are introducing in 0.3.0 as well.

Reply:
Thanks, I will take a look into the context command.